WolvenKit.App/ViewModels/Shell/ChunkViewModel.cs

```diff
@@ -3416,8 +3416,22 @@ private bool CreateArray()
     /// <returns>ChunkViewModel or null</returns>
     public ChunkViewModel? GetChildNode(int index)
     {
-        if (!IsArray || Properties is not ObservableCollectionExtended<ChunkViewModel> children ||
-            index >= children.Count)
+        if (!IsArray)
+        {
+            return null;
+        }
+
+        if (Properties is not ObservableCollectionExtended<ChunkViewModel> children)
+        {
+            return null;
+        }
+
+        if (Properties.Count == 0)
```

Comment:
Alright, let's now just move this after the null checks

Reply:
Can't do that - if properties isn't null but has the length 0, CalculateProperties will populate it. (Unless there really are no properties)

Reply:
No I meant move the calculate properties after the children index check

Reply:
Can you please suggest a change? I think we're talking past each other

Reply:
I meant this